src/test/java/ValidUrlsTest.java

```diff
@@ -0,0 +1,90 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.TestInstance.Lifecycle.PER_CLASS;
+
+import io.jenkins.plugins.designlibrary.UISample;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+import org.htmlunit.html.DomNode;
+import org.htmlunit.html.HtmlAnchor;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+
+@WithJenkins
+@TestInstance(PER_CLASS)
+class ValidUrlsTest {
```

Comment:
is this more ValidRelativeUrls?

It would be also cool to check if all URLs from the design library get a 200 response back as well?

Reply:
Yep - renamed.

Considered that yeah, worried it could be a little too chatty?

Reply:
Fairly common in static sites, e.g. jenkins.io does it:

https://github.com/jenkins-infra/jenkins.io/blob/master/Makefile#L27-L28

I've seen it elsewhere too.

Ideally not something enabled by default as I don't really like tests hitting the network if possibly, maybe it could only run in CI?

Reply:
Sweet, I'll create an issue from that.